js-green-licenses.json

```diff
@@ -1,5 +1,6 @@
 {
   "packageWhitelist": [
-    "log-driver"  // https://github.com/googleapis/nodejs-common/issues/16
+    "log-driver",  // https://github.com/googleapis/nodejs-common/issues/16
+    "escape-regexp-component"
```

Comment:
Can you add a comment describing why this should be whitelisted?

Reply:
Sorry, I thought I did.  I'll add one.

Reply:
Done